aten/src/ATen/core/function_schema.h

```diff
@@ -188,6 +190,13 @@ struct FunctionSchema {
   // arguments are not checked by schema
   bool is_vararg_;
   bool is_varret_;
+
+  // if no alias information is directly specified, what kind of "default"
+  // alias information should we infer?
+  // NB: due to alias analysis kind merging, this may be nullopt.  Eventually
+  // this should always be set no matter what
```

Comment:
Eventually, this should not be "always set" but I would argue it should go away entirely since this information is already fully representable in the `alias_info_` member. No action on this PR though.

Reply:
Yeah I think I agree with this.